src/main/java/com/buddy/api/domains/adoption/entities/AdoptionStatusHistoryEntity.java

```diff
@@ -0,0 +1,71 @@
+package com.buddy.api.domains.adoption.entities;
+
+import com.buddy.api.domains.adoption.enums.AdoptionStatus;
+import com.buddy.api.domains.shelter.entities.ShelterMemberEntity;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+@Data
+@Builder
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+@Table(name = "adoption_status_history")
+public class AdoptionStatusHistoryEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @EqualsAndHashCode.Include
+    @Column(name = "status_id", nullable = false, unique = true)
+    private UUID statusId;
```

Comment:
```suggestion
    @Column(name = "status_id", nullable = false, unique = true)
    private UUID statusId;
```

Mesma coisa referente à nomenclatura dos IDs.